### PR TITLE
Drop additional road props for z<=14

### PR DESCRIPTION
**Pull request by rmarianski**
@zerebubuth, @nvkelso could you review please?

### CODE REVIEW

File: queries.yaml [hunk 1]
```diff
@@ -568,13 +568,13 @@ post_process:
       where: >-
         (kind == 'path' and zoom < 15) or
         (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
-  # drop all oneway tags below zoom 14 (i.e: zoom >= 15) for all types.
+  # drop certain road properties below zoom 14 (i.e: zoom >= 15) for all types.
```
Comment (zerebubuth) [inline queries.yaml]: The "(i.e: zoom >= 15)" bit confuses me, should we take that out?

**Comment by zerebubuth** [comment]
:+1:

File: queries.yaml (hunk 1 above)
Comment (rmarianski) [inline queries.yaml]: Yea, good call. Updated in f123077.

**Comment by rmarianski** [comment]
Connects to https://github.com/tilezen/vector-datasource/issues/993

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**